starterkits/basic/packages/primary-components/molecules/checkbox/_m-checkbox.css

```diff
@@ -0,0 +1,40 @@
+%m-checkbox {
+  @apply flex flex-wrap;
+
+  &__original {
+    @apply absolute w-5 h-5 opacity-0;
+  }
+
+  &__custom {
```

Comment:
as far as i see they are(checkbox and radio) uses same layout. 
maybe it is possible to merge them? to common `control` component?

Reply:
maybe name that `__custom` element to something more clear?

Reply:
moved checkbox and radio components to atoms folder.
about "it is possible to merge them?" - i think better to keep it separately, since it's different a bit.
about "maybe name that __custom element to smth more clear?" - let's keep it as it is now. Idea here is that original square/circle is hidden, but "custom" is shown.